packages/cli/src/ui/components/messages/ToolConfirmationMessage.tsx

```diff
@@ -139,7 +139,7 @@ export const ToolConfirmationMessage: React.FC<
         value: ToolConfirmationOutcome.ProceedOnce,
       },
       {
-        label: `Yes, allow always "${executionProps.rootCommand} ..."`,
```

Comment:
this is unrelated to keyboard shortcuts. please move this to a separate pull request as it is unrelatedto the keyboard shortcut feature.